Resources/Textures/SS220/Clothing/Mask/Blue-shield-mask.rsi/meta.json

```
@@ -0,0 +1,27 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "taken from Paradise SS220 at https://github.com/ss220-space/Paradise/blob/f04540b212de5bf1919f7f1a52a7e20f0892a00e/icons/mob/clothing/neck.dmi, modified by gofinodzo (Discord) for SS220 SS14",
```

```suggestion
    "copyright": "taken from Paradise SS220 at https://github.com/ss220-space/Paradise/blob/f04540b212de5bf1919f7f1a52a7e20f0892a00e/icons/mob/clothing/????????????.dmi, modified by gofinodzo (Discord) for SS220 SS14",
```
вместо вопросиков отрой файл с маской и вставь туда блинб